README.rst

```diff
@@ -1,6 +1,13 @@
 
-.. image:: https://github.com/ansys/pymapdl/raw/main/doc/source/_static/logo_light.png
-   :alt: PyMAPDL
+.. raw:: html
+   
+   <p align="center">
+      <picture>
+         <source media="(prefers-color-scheme: dark)" srcset="https://github.com/ansys/pymapdl/blob/main/doc/source/_static/logo_dark.png">
+         <source media="(prefers-color-scheme: light)" srcset="https://github.com/ansys/pymapdl/blob/main/doc/source/_static/logo_light.png">
+         <img alt="PyMAPDL Logo" src="https://github.com/ansys/pymapdl/blob/main/doc/source/_static/logo_light.png">
```

Comment:
```suggestion
         <img alt="PyMAPDL Logo">
```
adding again the source can override the image all time with white logo. feel free to ignore this if you think its irrelevant

Reply:
@germa89